sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -81,6 +86,15 @@ namespace Azure { namespace Core { namespace Test {
     // This is a helper method to allow testing of the underlying operation<T> behaviors
     //  ClientOperations would not expose a way to control status
     void SetOperationStatus(OperationStatus status) { m_status = status; }
+
+    static StringOperation CreateFromResumeToken(
+        StringClient const& client,
+        std::string const& resumeToken)
```

Comment:
swap the order of these parameters

Reply:
why?  I find it more relevant to start by asking the client

Reply:
nevermind, I don't really care about the order hahahah, let's do whatever

Reply:
.NET `Operation` ctors have resume token first, and that's what you had shared in the code sample within the issue. Also, resume token is in the name, so that seems to be the more important piece.

Reply:
We don't have any way to create a contract for this. We rely on PR only.  Otherwise, each Operation is free to set the order.